src/pages/bootcamp-dimitar/index.tsx

```diff
@@ -0,0 +1,214 @@
+              <PageviewIcon />
+            </IconButton>
+            <Link href={`/bootcamp-dimitar/${cellValues.row.id}/edit`}>
+              <IconButton size="small" sx={{ mr: 1 }}>
+                <EditIcon />
+              </IconButton>
+            </Link>
+            <IconButton size="small" sx={{ mr: 1 }} onClick={handleDeleteModalOpen(cellValues.row)}>
+              <DeleteIcon />
+            </IconButton>
+          </>
+        )
+      },
+      minWidth: 150,
+    },
+  ]
+
+  return (
+    <CustomLayout>
+      <h1>All bootcampers</h1>
+      <IconButton aria-label="delete" href="/bootcamp-dimitar/create">
+        <PersonAddIcon />
+      </IconButton>
+      <Button
+        variant="contained"
+        onClick={() => setIsDeleteSelectedModalOpen(true)}
+        disabled={selectedRows.length == 0}>
+        Delete selected
+      </Button>
+      <DataGrid
+        rows={data || []}
+        columns={columns}
+        pageSize={5}
+        autoHeight
+        autoPageSize
+        checkboxSelection
+        disableSelectionOnClick
+        selectionModel={selectionModel}
+        onSelectionModelChange={(ids) => {
+          setSelectionModel(ids)
+          const selectedIDs = new Set(ids)
+          const selectedRows = data.filter((row) => selectedIDs.has(row.id))
+          setSelectedRows(selectedRows)
+        }}
+      />
+      <Modal
+        open={open}
+        onClose={handleClose}
+        aria-labelledby="modal-modal-title"
+        aria-describedby="modal-modal-description">
+        <Box sx={style}>
+          <Typography id="modal-modal-title" variant="h6" component="h2">
+            {row?.firstName} {row?.lastName}
+          </Typography>
+          <Typography id="modal-modal-description" sx={{ mt: 2 }}>
+            {row?.company}
+          </Typography>
+        </Box>
+      </Modal>
+      <Modal
+        open={isDeleteModalOpen}
+        onClose={handleDeleteModalClose}
+        aria-labelledby="modal-modal-title"
+        aria-describedby="modal-modal-description">
+        <Box sx={style}>
+          <Typography id="modal-modal-title" variant="h6" component="h2">
+            Delete?
+          </Typography>
+          <Typography id="modal-modal-description" sx={{ mt: 2 }}>
+            <Button variant="contained" onClick={deleteHandler}>
+              Yes
+            </Button>
+            <Button variant="contained" onClick={handleDeleteModalClose}>
+              No
+            </Button>
+          </Typography>
+        </Box>
+      </Modal>
+      <DeleteSelectedModal
+        isOpen={isDeleteSelectedModalOpen}
+        deleteHandler={deleteAllHandler}
+        handleDeleteModalClose={closeDeleteSelectedHandler}></DeleteSelectedModal>
+    </CustomLayout>
+  )
+}
+
+export const getServerSideProps: GetServerSideProps = async ({ locale }) => ({
+  props: {
+    ...(await serverSideTranslations(locale ?? 'bg', ['common'])),
+  },
+})
+
+export default BootcampDimitarList
```

Comment:
The convention of the project is to place the page component inside `components/` folder instead of in `pages/` folder and just reference it. In this way you can use the same component on multiple pages.

It's really hard to reuse the component when it's defined in `pages/` directory as it brings other artefacts together with it.